recipes/makefun/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "makefun" %}
+{% set version = "1.9.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/makefun-{{ version }}.tar.gz
+  sha256: 65234d5174316e2284e7e61a0ab2f2bfece88dbc0e14bd21e08db228462f503f
+
+build:
+  # Note: this is a universal python wheel wotking with python 2.7 and 3.5+
+  # but we cannot enable noarch since there is a dependency (funcsigs) that is python version specific.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  skip: True  # [py<35 and not py27]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python
+    - setuptools_scm
+  run:
+    - funcsigs  # [py<33]
```

Comment:
conda-forge does not build binaries for python<3.5, so I think you can remove this requirement and change the package to `noarch: python`.

Reply:
I should say, it doesn't build binaries for 2.7 by default. This can be turned on if you so desire once a feedstock is created:

https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-python-2-7

Reply:
If you decide that you just want to make the package for python>=3.5, then under `host` and `run` change `python` to `python >=3.5` when you switch to `noarch`.

Reply:
Thanks for this quick feedback ! I would like to also build the 2.7 version. So if I understand your answer, I should leave this PR "as is" for now, and later use the "conda admin please add python 2.7" once the dedicated repo is created. Is that correct ?

Reply:
I'd strongly advise against building 2.7 packages. The rest of the package ecosystem is no longer building packages for it

Reply:
As you prefer. The package is available on pypi anyway so python 2 users can still get it from there. By the way this calls for the more general question "why do all conda packages need to have *all* of their dependencies in conda as well (and can not reuse those in pypi)". If you have a link to the answer I would appreciate :)

I'll update the meta.yaml file now.

Reply:
done